src/Microsoft.OpenApi/Extensions/StringExtensions.cs

```diff
@@ -35,7 +36,23 @@ public static T GetEnumFromDisplayName<T>(this string displayName)
                 }
             }
 
-            return default(T);
+            return default;
         }
+
+        /// <summary>
+        /// Capitalizes each letter of a word in a string.
+        /// </summary>
+        /// <param name="input">String containing the words to be capitalized, delimited by the '-' character.</param>
+        /// <returns>String value with each word capitalized and concatenated.</returns>
+        public static string ToPascalCase(this string input)
+            => string.IsNullOrEmpty(input) ? input : string.Join(null, input.Split(new char[] { '-' }, StringSplitOptions.RemoveEmptyEntries)
```

Comment:
NIT: wrap on multiple lines, so the ternary is easier to read.

Reply:
Resolved